### ISSUE TITLE
Error setting up entry

### ISSUE THREAD

**Issue by shelasmax** [AlexxIT/YandexStation#187]
Добрый день.

Не работает интеграция несколько дней.
Много раз перезагружал HA. Не помогло.

```
Эта ошибка возникла в кастомной интеграции.

Logger: homeassistant.config_entries
Source: custom_components/yandex_station/core/yandex_session.py:200
Integration: Yandex.Station (documentation, issues)
First occurred: 26 сентября 2021 г., 23:36:26 (1 occurrences)
Last logged: 26 сентября 2021 г., 23:36:26

Error setting up entry **ххх** for yandex_station
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/aiohttp/connector.py", line 969, in _wrap_create_connection
    return await self._loop.create_connection(*args, **kwargs)  # type: ignore  # noqa
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 1056, in create_connection
    raise exceptions[0]
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 1041, in create_connection
    sock = await self._connect_sock(
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 955, in _connect_sock
    await self.sock_connect(sock, address)
  File "/usr/local/lib/python3.9/asyncio/selector_events.py", line 502, in sock_connect
    return await fut
  File "/usr/local/lib/python3.9/asyncio/selector_events.py", line 537, in _sock_connect_cb
    raise OSError(err, f'Connect call failed {address}')
OSError: [Errno 101] Connect call failed ('2a02:6b8::115', 443, 0, 0)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 304, in async_setup
    result = await component.async_setup_entry(hass, self)  # type: ignore
  File "/config/custom_components/yandex_station/__init__.py", line 79, in async_setup_entry
    if not await yandex.refresh_cookies():
  File "/config/custom_components/yandex_station/core/yandex_session.py", line 226, in refresh_cookies
    ok = await self.login_token(self.x_token)
  File "/config/custom_components/yandex_station/core/yandex_session.py", line 200, in login_token
    r = await self.session.post(
  File "/usr/local/lib/python3.9/site-packages/aiohttp/client.py", line 520, in _request
    conn = await self._connector.connect(
  File "/usr/local/lib/python3.9/site-packages/aiohttp/connector.py", line 535, in connect
    proto = await self._create_connection(req, traces, timeout)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/connector.py", line 892, in _create_connection
    _, proto = await self._create_direct_connection(req, traces, timeout)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/connector.py", line 1051, in _create_direct_connection
    raise last_exc
  File "/usr/local/lib/python3.9/site-packages/aiohttp/connector.py", line 1020, in _create_direct_connection
    transp, proto = await self._wrap_create_connection(
  File "/usr/local/lib/python3.9/site-packages/aiohttp/connector.py", line 975, in _wrap_create_connection
    raise client_error(req.connection_key, exc) from exc
aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host mobileproxy.passport.yandex.net:443 ssl:default [Connect call failed ('2a02:6b8::115', 443, 0, 0)]
```

**Comment by Vasiliy181** [comment]
У меня тоже перестали отображаться мини станция и лайт(

**Comment by AlexxIT** [comment]
Должно быть лучше в версии интеграции 3.6

**Comment by MisikAC** [comment]
Так же не запускается интеграция. НА версии 2024.1.0
НА пишет: Не удалось настроить 
Ошибка в журнале следующая:
Error setting up entry o*******z for yandex_station
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 406, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/yandex_station/__init__.py", line 137, in async_setup_entry
    await quasar.init()
  File "/config/custom_components/yandex_station/core/yandex_quasar.py", line 82, in init
    r = await self.session.get(f"{URL_V3_USER}/devices")
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/yandex_station/core/yandex_session.py", line 467, in get
    return await self._request("get", url, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/yandex_station/core/yandex_session.py", line 516, in _request
    return await self._request(method, url, retry - 1, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/yandex_station/core/yandex_session.py", line 516, in _request
    return await self._request(method, url, retry - 1, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/yandex_station/core/yandex_session.py", line 518, in _request
    raise Exception(f"{url} return {r.status} status")
Exception: https://iot.quasar.yandex.ru/m/v3/user/devices return 401 status

**Comment by kozyrev135-cloud** [comment]
здравствуйте 
не загружается интеграция 
что можно сделать
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
services-up: info: copying legacy longrun home-assistant (no readiness notification)
s6-rc: info: service legacy-services successfully started
2025-11-27 17:20:49.814 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration yandex_station which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2025-11-27 17:20:49.815 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration yandex_smart_home which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2025-11-27 17:20:49.815 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2025-11-27 17:20:49.816 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration alarmo which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2025-11-27 17:20:49.816 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration gismeteo which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2025-11-27 17:20:55.995 WARNING (MainThread) [aioesphomeapi.reconnect_logic] Can't connect to ESPHome API for home-assistant-voice-0933c6 @ 172.16.1.169: Error connecting to [AddrInfo(family=<AddressFamily.AF_INET: 2>, type=<SocketKind.SOCK_STREAM: 1>, proto=6, sockaddr=IPv4Sockaddr(address='172.16.1.169', port=6053))]: [Errno 101] Connect call failed ('172.16.1.169', 6053) (SocketAPIError)
2025-11-27 17:20:56.574 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.576 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_2 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.578 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_2 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.580 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_2 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.581 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_2 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.583 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_2 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.590 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_2 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.591 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_2 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.595 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_2 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.597 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_3 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.601 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_3 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.603 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_3 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.604 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_4 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.606 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_5 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.607 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_5 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.609 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_5 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.611 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_5 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.612 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_5 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.613 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_5 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.615 WARNING (MainThread) [homeassistant.components.mqtt.entity] The configuration for entity binary_sensor.zigbee2mqtt_bridge_connection_state_5 uses the deprecated option `object_id` to set the default entity id. Replace the `"object_id": "zigbee2mqtt_bridge_connection_state"` option with `"default_entity_id": "binary_sensor.zigbee2mqtt_bridge_connection_state"` in your published discovery configuration to fix this issue, or contact the maintainer of Zigbee2MQTT that published this config to fix this. This will stop working in Home Assistant Core 2026.4
2025-11-27 17:20:56.616 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry kozyrev.fed for yandex_station
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 761, in __async_setup_with_context
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/yandex_station/__init__.py", line 164, in async_setup_entry
    await quasar.init()
  File "/config/custom_components/yandex_station/core/yandex_quasar.py", line 222, in init
    await self.load_speakers()
  File "/config/custom_components/yandex_station/core/yandex_quasar.py", line 248, in load_speakers
    else await self.add_scenario(device_id, hash)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/yandex_station/core/yandex_quasar.py", line 302, in add_scenario
    assert resp["status"] == "ok", resp
           ^^^^^^^^^^^^^^^^^^^^^^
AssertionError: {'request_id': '06644fc7-09b3-470b-9a27-66b368ba7967', 'status': 'error', 'code': 'NAME_IS_ALREADY_TAKEN', 'message': 'Данное имя уже используется'}
2025-11-27 17:20:59.292 WARNING (MainThread) [homeassistant.components.onvif] Dahua: No registered handler for event from f8:ce:07:35:1a:3c: {
    'SubscriptionReference': {
        'Address': {
            '_value_1': 'http://192.168.1.194/onvif/Subscription?Idx=21',
            '_attr_1': None
        },
        'ReferenceParameters': None,
        'Metadata': None,
        '_value_1': None,
        '_attr_1': None
    },
    'Topic': {
        '_value_1': 'tns1:VideoSource/tns1:Device/Trigger/DigitalInput',
        'Dialect': 'http://www.onvif.org/ver10/tev/topicExpression/ConcreteSet',
        '_attr_1': {
    }
    },
    'ProducerReference': None,
    'Message': {
        '_value_1': {
            'Source': {
                'SimpleItem': [
                    {
                        'Name': 'InputToken',
                        'Value': '00000'
                    }
                ],
                'ElementItem': [],
                'Extension': None,
                '_attr_1': None
            },
            'Key': None,
            'Data': {
                'SimpleItem': [
                    {
                        'Name': 'LogicalState',
                        'Value': 'false'
                    }
                ],
                'ElementItem': [],
                'Extension': None,
                '_attr_1': None
            },
            'Extension': None,
            'UtcTime': datetime.datetime(2025, 11, 27, 14, 20, 57, tzinfo=datetime.timezone.utc),
            'PropertyOperation': 'Initialized',
            '_attr_1': {
        }
        }
    }
}
2025-11-27 17:23:21.890 ERROR (MainThread) [aiohttp.server] Error handling request from 192.168.1.150
